sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/LocalModelFetcher.cs

```diff
@@ -65,7 +69,9 @@ public FetchResult Fetch(string dtmi, Uri repositoryUri, CancellationToken cance
                     fnfError = string.Format(CultureInfo.CurrentCulture, ServiceStrings.ErrorFetchingModelContent, tryContentPath);
                 }
 
-                throw new FileNotFoundException(fnfError);
+                throw new RequestFailedException(
+                    $"{string.Format(CultureInfo.CurrentCulture, ServiceStrings.GenericResolverError, dtmi)} {fnfError}",
```

Comment:
Should instead use `FormattableString.CurrentCulture`.

<https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring.currentculture?view=net-5.0#System_FormattableString_CurrentCulture_System_FormattableString_>

Reply:
Looks cool. Investigated a bit and it appears its not supported in netstandard2.1. Seems its not viable unless we want to apply preprocessor directives.

Reply:
You are right. Bummer.

I do not care for pre-processor directives, so I agree we should leave it be.